torch/_inductor/codegen/cpp_bmm_template.py

```diff
@@ -180,6 +180,8 @@ def get_options(
         )
 
         BX, BW, BY = options["X"], options["W"], options["Y"]
+        # Create the BMM size_vars in the kernel
+        kernel.size(BY, 0)
```

Comment:
Why we invoke this line but didn't use the return value?

Reply:
When we create a new sympy Symbol (`b_index`) which we use to index into the GEMM problem, it adds a new sizevar to the CppTemplateKernel. However, there is some underlying code which is dependent on the ordering of sizevars in the KernelArgs object. So calling `kernel.size(BY, 0)` causes all the sizevars present in BY to be saved to the kernel sizevars, so they are in the proper order and numbering. Then we can add the `b_index` variable without any issues.

Reply:
This seems a bit hacky, is there a more general solution?

Reply:
I've updated how we set sizevars with a new method that is called directly, instead of relying on the kernel.size method.